views/publications-details.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/two-columns.html" %}
+
+{% block main %}
+<article>
+  <h2>Publication description</h2>
+  <ul class="overview-list">
+    <li>{{ publication.title }}</li>
+  </ul>
+
+  <h2>Source</h2>
+  <a href="{{publication.source_url}}">{{publication.source.name}}</a>
```

Comment:
Sorry, I haven't ran the code so I can't answer it. Where does `publication.source_url` points to?